src/server/adminJs/tabs/projectFraudTab.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  canAccessProjectStatusReasonAction,
+  ResourceActions,
+} from '../adminJsPermissions';
+import {
+  AdminJsContextInterface,
+  AdminJsRequestInterface,
+} from '../adminJs-types';
+import { logger } from '../../../utils/logger';
+import csv from 'csvtojson';
+import { messages } from '../../../utils/messages';
+import { ProjectFraud } from '../../../entities/projectFraud';
+
+export const createProjectFraud = async (
+  request: AdminJsRequestInterface,
+  response,
+  context: AdminJsContextInterface,
+) => {
+  let message = messages.PROJECT_FRAUD_HAS_BEEN_CREATED_SUCCESSFULLY;
+  logger.debug('createProjectFraud has been called() ', request.payload);
+
+  let type = 'success';
+  try {
+    const { projectId, qfRoundId, csvData, confirmedFraud } = request.payload;
+    if (csvData) {
+      // Parse the CSV data
+      const jsonArray = await csv().fromString(csvData);
+
+      // Validate and extract all unique walletAddresses
+      const slugs: string[] = [];
+      for (const obj of jsonArray) {
+        if (!obj.walletAddress || !obj.qfRoundId) {
+          throw new Error('Missing slug for project');
```

Comment:
is it possible to log projectID in this error log

Reply:
Thanks I made a mistake here ill log it correctly.

Reply:
Since the value is missing, we are exporting a csv with qfroundId, slug
I will log instead the ROW of the csv that is missing. So the user knows which is wrong.

Reply:
I used slugs because comms team use it regularly